src/index.js

```diff
@@ -64,9 +61,7 @@ async function initStream() {
     data: {
       // This designates which subscription will be used to filter data to the firehose
       subscriptionId: process.env.FIREHOSE_SUBSCRIPTION_ID,
-    },
-    params: {
-      keep_alive: program.opts().keepAlive,
+      start: process.env.START,
```

Comment:
**issue:** This will always reconnect to whatever start value is passed in leading to duplicate messages. What we want to do is keep `start` up-to-date with the last Id seen. 

Something like:
```
let start = program.opts.start

async function listen() {
  const stream = await initStream();
  stream
    .on('data', (chunk) => {
        const data = JSON.parse(chunk.toString());
        start = data.id
        })
 }

async function initStream() {
...blahblahblah...
   params: { start }
}[

```